src/app/lib/analyticsUtils/index.js

```diff
@@ -126,12 +127,20 @@ export const getReferrer = (platform, origin, previousPath) => {
 };
 
 export const getAtUserId = () => {
-  if (onClient()) {
-    const atuserid = Cookie.getJSON('atuserid');
-    return pathOr(null, ['val'], atuserid);
+  if (!onClient()) return null;
+
+  const cookieName = 'atuserid';
+  const cookie = Cookie.getJSON(cookieName);
+  let val = pathOr(null, ['val'], cookie);
+  const expires = 395; // expires in 13 months (365 + 30 days)
```

Comment:
What is the significance of the number of days? It could be 365+28 for example, is for compatibility with Orbit or an ATI requirement?

Reply:
JS Cookie uses days when you pass a number. Compatibility wise, I don't really know. The issue said cookie should expire in 13 months. I went with `365+30` assuming 13 months was an approximation, hence a simpler implementation that calculating and passing a date in the future.

Reply:
>  I went with 365+30 assuming 13 months was an approximation, hence a simpler implementation than calculating and passing a date.
That makes sense @samora, @jimjohnsonrollings what is the significance of the 13 month requirement? Does it need to be precise or an approximate calculation of 13 months (e.g. 365 + 30) ? Just wondering if:
- this is necassary for GDPR compliance
- and/or necassary to synchronise (and not conflict) with Orbit

Reply:
My instinct is that it's more about the synchronising with Orbit.  Any idea what their code uses to pick the date?

Reply:
I've had a look in https://github.com/bbc/nav-orbit-webmodules and https://github.com/bbc/nav-orbit and can't find any code setting `atuserid`, I think it may actually be part of `reverb` that Orbit delivers? I'd suggest someone ask in #help-reverb to clarify the reason for the 13 month~ expiry?

Reply:
I'm told that it's `397 * 86400` seconds - but have asked ATI directly as they set it for Orbit.

Reply:
Confirmed.  397 days

Reply:
Cool, change to 397 and I can give this a :white_check_mark: